ibis/sql/sqlite/client.py

```diff
@@ -48,6 +49,36 @@ def udaf(f):
     return f
 
 
+@udf
+def _ibis_sqlite_reverse(string):
+    if string is not None:
```

Comment:
Follow up here is to wrap these in another function call that checks all arguments for NULL and if any are NULL then return a NULL, otherwise call the function. This will clean up some code a bit.